dali_mqtt_daemon.py

```diff
@@ -223,31 +200,44 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
         ).group(1)
     )
     try:
-        level = int(msg.payload.decode("utf-8"))
-        if not 0 <= level <= 255:
-            raise ValueError
-        logger.debug("Set light <%s> brightness to %s", light, level)
-        data_object["driver"].send(gear.DAPC(address.Short(light), level))
-        if level == 0:
-            # 0 in DALI is turn off with fade out
-            mqtt_client.publish(
-                MQTT_STATE_TOPIC.format(data_object["base_topic"], light),
-                MQTT_PAYLOAD_OFF,
-                retain=True,
+        if light not in data_object["all_lamps"]:
+            raise KeyError
+        lamp_object = data_object["all_lamps"][light]
+        try:
+            level = int(msg.payload.decode("utf-8"))
+            lamp_object.level = level
+            logger.debug("Set light <%s> brightness to %s", light, lamp_object.level)
```

Comment:
this debug line should be in the Lamp class setter

Reply:
But you realize that this log message will be trigger also due initialization and always after correctly update `level`? I mean, there will more times with this message than currently

Reply:
Well the idea of the lamp_object is for it to be in sync (a digital-twin) of the physical lamp.

Reply:
So maybe I should real set brightness in setter by `send` method of driver object?

Reply:
but try not to create to much coupling

Reply:
Done. look on new version. Changing brightness level is in lamp class